glide-core/Cargo.toml

```diff
@@ -30,6 +30,7 @@ nanoid = "0.4.0"
 
 [features]
 socket-layer = ["directories", "integer-encoding", "num_cpus", "protobuf", "tokio-util"]
+standalone_heartbeat = []
```

Comment:
What is the Idea of this feature?

Reply:
it's a standalone feature to run heartbeats on the connections which we decided not to enable yet, this is why it's behind a feature flag.